pdr_backend/pdr_dashboard/pages/home.py

```diff
@@ -286,3 +301,42 @@ def get_table(self, table_id, columns, data):
                 ),
             ],
         )
+
+    def getPredictoorsConfigModal(self):
+        return dbc.Modal(
+            html.Div(
+                [
+                    html.H2("Configure Favourite Predictoor Addresses"),
+                    html.P(
+                        """Add predictoor addresses to be saved 
+                        and automatically selected when the app opens.""",
+                        style={"marginBottom": 0},
+                    ),
+                    html.P("Enter one address per line."),
```

Comment:
https://dash.plotly.com/dash-html-components/datalist could also be an option

Reply:
from what I see that is mainly used as a dropdown select